classes/ClaytonTestClass1.cls

```diff
@@ -0,0 +1,953 @@
+public class ClaytonTestClass1 {
+    public ClaytonTestClass1(){
+        List<Case> cList = [Select Id,CaseNumber, AccountId, 
```

Comment:
If the table in this query contains more than 50'000 records a runtime exception will be thrown. Consider either introducing a `WHERE` clause, or adding a `LIMIT` to contain the number of records returned.